.gitignore

```diff
@@ -1,7 +1,4 @@
 # IntelliJ
-.idea/
```

Comment:
@KristinaKots , why did you remove my changes?

Reply:
There were too much same expressions.

Reply:
Ok. I get it. My fault... 
Thanks.